Makefile

```diff
@@ -20,6 +20,15 @@ endif
 PROGRAM ?= hello
 TARGET ?= sifive-hifive1
 
+# Coremark require PORT_DIR set for different OS, freedom-metal for us!
+ifeq ($(PORT_DIR),)
+#$(error PORT_DIR not set for $(PROGRAM))
+ifeq ($(PROGRAM),coremark)
+PORT_DIR = freedom-metal
+#$(error PORT_DIR set to $(PORT_DIR), for $(PROGRAM))
```

Comment:
These lines are still commented out.

Reply:
Updated the change, that should be for debugging purpose. We don't need it now.

Reply:
I've sent a PR to freedom-devicetree-tools.